lua/lspconfig/server_configurations/eslint.lua

```diff
@@ -160,7 +160,7 @@ npm i -g vscode-langservers-extracted
 
 vscode-eslint-language-server provides an EslintFixAll command that can be used to format document on save
 ```vim
-autocmd BufWritePre <buffer> <cmd>EslintFixAll<CR>
+autocmd BufWritePre <buffer> EslintFixAll
```

Comment:
I think it would be better to add `silent!`, so that the autocmd would fail silently if the ESLint-Server is not running and therefore `EslintFixAll` is not aviable.

Line would look like this
```
autocmd BufWritePre <buffer> silent! EslintFixAll
```